doc/releases/changelog-0.39.0.md

```diff
@@ -394,6 +394,9 @@
 
 <h3>Bug fixes 🐛</h3>
 
+* The wire order for `Snapshot`'s now matches the wire order of the device, rather than the simulation.
```

Comment:
Just out of curiosity, what is the expected behaviour if the device wasn't created with wires?

Reply:
"Expected" can always be a tricky one, but I think if the device wasn't created with wires we have the flexibility to do things with whatever wire order we want.